python/cudf/cudf/core/series.py

```diff
@@ -856,7 +856,9 @@ def to_frame(self, name=None):
 
     @_cudf_nvtx_annotate
     def memory_usage(self, index=True, deep=False):
-        return sum(super().memory_usage(index, deep).values())
+        return self._column.memory_usage + (
+            self.index.memory_usage() if index else 0
```

Comment:
Same comment as in DataFrame about accessing `_index` instead of `index`
```suggestion
            self._index.memory_usage() if index else 0
```